docs/src/test-components-js.md

```diff
@@ -218,7 +218,7 @@ beforeMount(async ({ hooksConfig }) => {
 ```js
 // src/Users.spec.tsx
 import { test, expect } from "@playwright/experimental-ct-react";
-import React from "react";
+import * as React from "react";
```

Comment:
Why did this change? Does React 18 break the short import style? That's something TSC should be able to handle based on its settings.

Reply:
This is a drive-by change since the recommended way to import has been `import *` for a while now.

Reply:
Could you point to where it is recommended?

Reply:
There's no explicit blogpost about it but the official docs no longer use default imports. It's all named imports (https://beta.reactjs.org/) i.e. no default import anymore.

Reply:
@pavelfeldman here is great blog post from @kentcdodds https://epicreact.dev/importing-react-through-the-ages/ IMHO both versions are fine.